python_modules/dagster/dagster/_core/executor/multi_environment/remote_step_information.py

```diff
@@ -0,0 +1,39 @@
+from typing import NamedTuple, Optional
+
+from dagster._core.execution.plan.state import KnownExecutionState
+from dagster._core.execution.retries import RetryMode
+from dagster._core.instance import DagsterInstance
+from dagster._serdes.serdes import (
+    deserialize_as,
+    serialize_dagster_namedtuple,
+    whitelist_for_serdes,
+)
+
+
+@whitelist_for_serdes
```

Comment:
Give a man a KV store and Lord knows what he'll stash in there!

Reply:
definitely need to figure out some expirey/removal for this data if we are going to use the KV store for it so it doesn't pile up endlessly.

Reply:
@alangenfeld I think it is pretty safe to clean up on run completion. however we need a way to index into that...